doc/specs/thread_tagging.md

```diff
@@ -0,0 +1,215 @@
+# Thread Tagging
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+
+## Motivation
+
+The forums are a major component of the site, with a vast wealth of knowledge. 
+Having the ability to tag a thread would make searching much easier and more 
+streamlined, and to help keep certain forums on topic.
+
+## Business Rules
+
+* Any thread in a tagging-enabled forum must be able to be labeled and tagged by:
+  * The user creating the thread
+  * A user with the tag management permission enabled
+  * A user that belongs to a user class given tag editing permission on a per-forum basis
+* Labels and tags must be chosen from pre-approved lists. The list will be maintained by
+  users with the tag administration permission enabled.
+* Each tagging enabled forum must have unique label and tag lists.
```

Comment:
Went with "Each tagging enabled forums must have unique labels and tags." Resolved.